ui/dashboards/src/utils/functions.ts

```diff
@@ -1,16 +0,0 @@
-// Copyright 2023 The Perses Authors
```

Comment:
Renamed this utility for clarity, but I added enough changes that git treated it like a delete.